content/md/first-steps/manage-your-attributes.md

```diff
@@ -268,31 +268,97 @@ Unlike products you cannot restore or revert an attribute’s version.
+1.  Click on the trash can icon
+1.  Confirm the action in the popin
 
 **The attribute and all its relevant information will be deleted in Akeneo PIM. Once you have confirmed the action, it cannot be reverted.**
 
-::: ee
-An attribute that is used in a published product cannot be deleted. You first need to unpublish the product(s) and then delete the attribute.
+##Delete an attribute and keeping the related data
+
+If you need to change an attribute property (to make it scopable and/or localisable for instance), or if you want to change the attribute type (from a simple text to a text area for instance), you need to delete the attribute.  
+But before doing so, **if you plan to recreate another attribute with the same code as previously used**, and if you want to keep the data already entered in this attribute, you will need to export the data (first step), delete the attribute (second step), create another attribute (third step), then reimport the data back (fourth step).  
+To do it so, please follow this walkthrough:
+
+###First step: export the data
+
+:::warning
+Before starting this step, make sure you know how to configure a new export profile. If you need more help, please read [How to create a new product export profile](#product-export-builder.html#create-a-new-product-export-profile).
 :::
 
+1.  Go to `Exports` and click on `Create export profile`
+1.  Write a code and a label
+1.  Select the job in the `job` field, depending if you want to export products or product models (if you need to export both products and product models, you will need to create an export profile for each one)
+![Select the job to export your data](../img/Delete-Attribute_Job_Export-profile.png)
+1.  In the `Content`, filter the products by clicking on `Add attribute` and select the attribute you want to delete
+1.  Set its value to "Is not empty"  
+![Select the appropriate attribute](../img/Delete-Attribute_Job_Export-profile_Content.png)
+1. In `Global settings` tab, make sure the option `Export with labels` is not activated
+1. Check the different other options (channel, locales, format...) to refine your export and click on `Save`
+1.  Click on `Export now`
+
+###Second step: delete the attribute
+
+To delete the attribute, follow one of these two ways below:
+* Through the [attribute edit form](manage-your-attributes.html#through-the-attribute-edit-form)
+* Through the [delete shortcut](manage-your-attributes.html#through-the-delete-shortcut-in-the-grid) in the grid
+
 ::: warning
-If you cannot access the attribute page or if you cannot add/delete an attribute, it is probably because you do not have the permission to. Please refer to [Manage your user roles](build-your-user-roles.html) to learn about roles.
+If you plan to delete a simple or multi select attribute, and if you want to keep their data, you will also need to export the attribute options through the Akeneo default export profile named `Demo XLSX attribute option export`.
```

Comment:
What about PIM already installed where this demo export is no more ?
Do we need to expand on this ?

Reply:
Good point. Adding more documentation. Thanks :)